openlibrary/macros/AffiliateLinks.html

```diff
@@ -0,0 +1,34 @@
+$def with (opts)
+
+$ isbn = opts['isbn']
+$ asin = opts['asin']
+$ scribd = opts['scribd']
+
+$ alibris = "http://click.linksynergy.com/fs-bin/click?id=ECh45gHEoh4&offerid=99238.122856000&type=2&subid=0&tmpid=939&RD_PARM1=http%253A%252F%252Fwww.alibris.com/booksearch%253Fqsort%253Dp%2526isbn%253DXXX"
+
+$ amazon = "http://www.amazon.com/dp/XXX/?tag=internetarchi-20"
+
+$# AbeBooks target link: http://www.abebooks.com/servlet/SearchResults?isbn=XXX
+$# AbeBooks affiliate tracking: http://affiliates.abebooks.com/c/64277/77798/2029?u=[ENCODED TARGET LINK]
+$ abebooks = "http://affiliates.abebooks.com/c/64277/77798/2029?u=http%3A%2F%2Fwww.abebooks.com%2Fservlet%2FSearchResults%3Fisbn%3DXXX"
+
+$ bibliocom = "http://www.biblio.com/editions.php?isbn=XXX&aid=openlibrary"
+
+$ bookdepository = "https://www.bookdepository.com/search?searchIsbn=XXX"
+
+$ powells = "http://www.powells.com/partner/44336/biblio/XXX"
+
+<ul>
+$if isbn or asin:
```

Comment:
Alternatively you could replace the initial check of `$if isbn or asin` by instead changing the final elif to: `if scribd and not (asin or isbn)`. This seems more logical as it reflects the constraint that whether to show scribd depends on the presence of isbn or asin -- not the isbn and asin blocks.